documentation/docs/02_account_manager.md

```diff
@@ -76,6 +77,11 @@ Initialise the wallet with a randomly generated mnemonic and a given node.
 $ ./wallet init --node "http://localhost:14265/"
 ```
 
+Initialise the wallet with a given coin type.
```

Comment:
Please add somewhere https://github.com/satoshilabs/slips/blob/master/slip-0044.md for the coin types and maybe also 4218=IOTA, 4219=Shimmer, because people not familiar with it wouldn't know what this number